index.js

```diff
@@ -1,19 +1,22 @@
-require ('newrelic');
+require('newrelic');
 const createScheduler = require('probot-scheduler');
 const apiForSheetsModule = require('./lib/apiForSheets');
 const checkMergeConflictsModule = require('./lib/checkMergeConflicts');
 const checkPullRequestLabelsModule = require('./lib/checkPullRequestLabels');
 const whitelistedAccounts = (
   (process.env.WHITELISTED_ACCOUNTS || '').toLowerCase().split(','));
-var pullRequestAuthor;
 
-module.exports = (robot) => {
-  scheduler = createScheduler(robot, {
+/**
+ * This is the main entrypoint to the Probot app
+ * @param {import('probot').Application} app
+ */
+module.exports = (app) => {

```

Comment:
Any specific reason for changing robot to the app? (Just wanted to know the context behind this)

Reply:
Not really.
That's what's used in the doc so I decided to do it.

Reply:
In that case, I would suggest undoing this change. Considering the main repo app name is `oppia` so we should name it similarly (either make it `oppiabot` (or `robot`))

Reply:
`oppiabot` will suffice